backend/Tasque.BLL/Services/ProjectService.cs

```diff
@@ -177,4 +164,29 @@ public async Task ChangeUserRole(ChangeUserRoleDto roleDto)
         _db.UserProjectRoles.Add(userProjecRole);
         await _db.SaveChangesAsync();
     }
+
+    public async Task<BoardInfoDto> GetProjectBoard(int projectId)
+    {
+        var project = await _db.Projects
+            .Include(x => x.Columns).ThenInclude(x => x.Tasks)
+            .Include(x => x.Users)
+            .FirstOrDefaultAsync(x => x.Id == projectId)
+            ?? throw new CustomNotFoundException("project");
+
+        return _mapper.Map<BoardInfoDto>(project);
+    }
+
+    public async Task<BoardInfoDto> UpdateBoard(BoardInfoDto board)
+    {
+        var proj = _db.Projects
+            .Include(x => x.Columns).ThenInclude(x => x.Tasks)
+            .Include(x => x.Users)
+            .FirstOrDefault(x => x.Id == board.Id)
+            ?? throw new CustomNotFoundException("project board");
+        var mapped = _mapper.Map(board, proj);
```

Comment:
![mapping exception](https://user-images.githubusercontent.com/91162284/189219145-77b46c78-f55e-4048-9518-b7effb52e8d6.png)
This row throws an exception on moving task card.